examples/config.json

```diff
@@ -268,27 +268,18 @@
 	    "cores"    : 72
     },
 
-    "ornl.summit"  : {
-        "project"  : "BIP178",
-        "queue"    : "batch",
-        "schema"   : "local",
-        "cores"    : 84,
-        "gpus"     : 12
-    },
-
     "local.summit" : {
         "project"  : "BIP178",
```

Comment:
This requires a costly work around from the user. E.g., sed -i 's/BIP178/geo111/g' [...]/lib/python2.7/site-packages/radical/pilot/configs/resource_ornl.json Do we have a path towards eliminating this hard coded allocation value? (also in #1904)

Reply:
At the moment we do not have the ability to fix that.  I don't like the fact that the examples depend on that configuration file in general, tbh - it makes it more difficult to run examples for end users, and as you observe, the data get out of sync anyway.  I think the main reason to have that config in place is *for us* to be able to easily run the examples, and to avoid the need to do the same changes in several files.

I see two approaches to alleviating the issue.  First, I think that some example configurations should become part of the resource config, like default access schema, number of cores to use for examples, etc.  Secondly, a user configuration file would make sense,  which lists preferred access schema for all resources, preferred queues, user names, allocations, etc.

Both are fairly straight forward to implement - but both require significant coding I think.  The configuration restructuring in v.2 will make this (somewhat) simpler though.